frontend/package.json

```diff
@@ -1,5 +1,4 @@
 {
-  "proxy": "http://localhost:3001",
```

Comment:
I just want to be sure that it's normal that you delete "proxy"

Reply:
yes it is! This proxy definition is too "hardcoded" for us, in particular from the point of view of the frontend container (trying to reach the backend), the backend is not at `https://localhost:3001` it is inside a dedicated docker network *(see the end of the `Dockerfile.dev` and the `network:` section under each container)* and thus needs to be proxied to the right location: `https://react-ap-dev:3001` configurable by environment variables!
This is done by [301f26c](https://github.com/iScsc/iscsc.fr/pull/86/commits/301f26c99bfa1721b0183fadbdd1a1530fd7e38e) at runtime 😉

Reply:
@Laudut don't forget to close this conversation!